### PR TITLE
-added code to dump environmental variables during the build process

### PR DESCRIPTION
**Pull request by raymondblanc**

### CODE REVIEW

File: cloudera-integration/csd/scripts/control.sh [hunk 1]
```diff
@@ -23,6 +23,17 @@ set -euo pipefail
 
 CMD=$1
 
+# For better debugging
```
Comment (afoerster) [inline cloudera-integration/csd/scripts/control.sh]: Can you describe what this is doing? "Better debugging" is vague

File: cloudera-integration/csd/scripts/control.sh [hunk 2]
```diff
@@ -23,6 +23,17 @@ set -euo pipefail
...
+date 1>&2
+
+CMD=$1
+
+function log {
```
Comment (afoerster) [inline cloudera-integration/csd/scripts/control.sh]: Nice.

File: cloudera-integration/csd/scripts/control.sh [hunk 3]
```diff
@@ -23,6 +23,17 @@ set -euo pipefail
...
+  timestamp=$(date)
+  echo "$timestamp: $1"       #stdout
+  #echo "$timestamp: $1" 1>&2; #stderr
```
Comment (bpmcd) [inline cloudera-integration/csd/scripts/control.sh]: You could delete the commented out line if it's not useful anymore.

**Comment by bpmcd** [comment]
Looks good to me.

**Pull request merged**